playbooks/rhv/inventory.example

```diff
@@ -0,0 +1,99 @@
+[all:vars]
+app_dns_prefix=apps
+public_hosted_zone=example.com
+load_balancer_hostname=lb0.{{public_hosted_zone}}
+openshift_master_cluster_hostname="{{ load_balancer_hostname }}"
+openshift_master_cluster_public_hostname=openshift-master.{{ public_hosted_zone }}
+openshift_master_default_subdomain="{{ app_dns_prefix }}.{{ public_hosted_zone }}"
+openshift_public_hostname="{{openshift_master_cluster_public_hostname}}"
+
+[OSEv3:children]
+nodes
+masters
+etcd
+lb
+
+[OSEv3:vars]
+# General variables
+ansible_ssh_user=root
+console_port=8443
+debug_level=2
+deployment_type=openshift-enterprise
+openshift_debug_level="{{ debug_level }}"
+openshift_deployment_type="{{ deployment_type }}"
+openshift_master_cluster_method=native
+openshift_node_debug_level="{{ node_debug_level | default(debug_level, true) }}"
+openshift_release='3.10'
+openshift_vers=v3_10
+
+# RHV Specific Settings
```

Comment:
I think this section should be added to https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.example

Reply:
It should probably be removed until we have a cloud provider ready to go on install.

Reply:
I think we should remove as much as we can and leave the oVirt / RHV specifics.
And reference the general variables to the "hosts.example" file for more options.
And if there are variables that are rhv specific variable with defaults that were added to the core openshift-playbooks(deploy_cluster, prerequisites), they should be added to the  "hosts.example" file.

Reply:
+1, I'll take a pass at doing that today.